app/controllers/sessions_controller.rb

```diff
@@ -21,6 +21,11 @@ def mfa_create
     end
   end
 
+  def destroy
```

Comment:
I doubt there is a need for this. We check if the user is signed in all owner methods we need to, adding this doesn't gain us anything.

Reply:
If I logout and login to some other user's account, it allows me to access without verifying. So, I had to delete the key.

Reply:
> logout and login to some other user's account

Note that the issue we are trying to mitigate is someone leaving their account logged in unintentionally for an extended period. If the user had just logged in, it is okay if we don't ask them for a password.
I understand it is a bit inconsistent but no security issue otherwise.